internal/containerizedengine/types.go

```diff
@@ -13,10 +13,7 @@ import (
 const (
 	containerdSockPath  = "/run/containerd/containerd.sock"
 	engineContainerName = "dockerd"
-	engineNamespace     = "docker"
-
-	// Used to signal the containerd-proxy if it should manage
-	proxyLabel = "com.docker/containerd-proxy.scope"
+	engineNamespace     = "com.docker"
```

Comment:
What's the context for this change?

Reply:
Hmm... not sure - I didn't write that part - let me poke around a bit.

Reply:
It's still used, and puts the image into the com.docker namespace on containerd.

Reply:
Not a big concern, but will this affect people that upgrade from beta1 to beta2 (if beta1 started the container in the old namespace)?